data-raw/region_codes.R

```diff
@@ -57,11 +57,63 @@ germany_codes <- tibble::tibble(
   )
 )
 
+# france codes
+
+france_level_1_codes <- tibble::tibble(
+  iso_code = c(
+    "FR-ARA", "FR-BFC", "FR-BRE", "FR-CVL", "FR-20R", "FR-GES", "FR-GP", "FR-GF", "FR-HDF", "FR-IDF", "FR-RE", "FR-MQ", "FR-YT",
+    "FR-NOR", "FR-NAQ", "FR-OCC", "FR-PDL", "FR-PAC", "FR-PM", "FR-BL", "FR-MF"
+  ),
+  insee_code = c("84", "27", "53", "24", "94", "44", "01", "03", "32", "11", "04", "02", "06", "28", "75", "76", "52", "93", "05", "07", "08"),
+  region_level_1 = c(
+    "Auvergne-Rhone-Alpes", "Bourgogne-Franche-Comte", "Bretagne", "Centre-Val de Loire", "Corse", "Grand-Est", "Guadeloupe",
+    "Guyane (francaise)", "Hauts-de-France", "Ile-de-France", "La Reunion", "Martinique", "Mayotte", "Normandie", "Nouvelle-Aquitaine",
+    "Occitanie", "Pays-de-la-Loire", "Provence-Alpes-Cote-d'Azur", "Saint-Barthelemy", "Saint-Martin", "Saint-Pierre et Miquelon"
+  )
+)
+
+# Get region names and codes
+france_level_2_codes_url <- "https://en.wikipedia.org/wiki/ISO_3166-2:FR"
```

Comment:
We are currently thinking about this but for now I think this much code could benefit from going in its own script in `data-raw` with the cleaned up region codes saved as a csv and then loaded in `region_codes` here.

Reply:
I think my latest commit should provide this.